diff --git a/dist/index.js b/dist/index.js
index 69f26e232559cb3b7aeccbda4830c9fbc63745f1..02bd696016cb7a16e71b73043b161e9f5f15f53f 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -13,7 +13,7 @@ class ProcessingError extends Error {}
 async function poll(check, interval, maxRetry, immediate = false) {
   let lastError = new Error('Polling skipped');
   for (let i = 0; i < maxRetry; i += 1) {
-    if (!immediate && !i) await delay(interval);
+    if (!immediate || i > 0) await delay(interval);
     try {
       return await check(i);
     } catch (err) {
