diff --git a/_root.js b/_root.js
index d2852bed4d2e469b3b87ed31df2350047e005ff4..6ea47914730240d0a3a02b0cab58a2622ffae456 100644
--- a/_root.js
+++ b/_root.js
@@ -4,6 +4,6 @@ var freeGlobal = require('./_freeGlobal');
 var freeSelf = typeof self == 'object' && self && self.Object === Object && self;
 
 /** Used as a reference to the global object. */
-var root = freeGlobal || freeSelf || Function('return this')();
+var root = freeGlobal || freeSelf || window;
 
 module.exports = root;
